### PR TITLE
Add healthCheck model and controller

### PR DESCRIPTION
**Pull request by vpchung**
With this PR:

* add healthCheck endpoint

I also moved the unit tests written for the models out of `integration/` and into a `unit/` folder.

### CODE REVIEW

File: server/openapi_server/openapi/openapi.yaml [hunk 1]
```diff
@@ -422,6 +424,27 @@ paths:
       tags:
       - Grant
       x-openapi-router-controller: openapi_server.controllers.grant_controller
+  /healthcheck:
+    get:
+      description: Get information about the health of the service
+      operationId: get_health_check
+      responses:
```
Comment (tschaffter) [inline server/openapi_server/openapi/openapi.yaml:431]: The 500 response is missing. https://github.com/nlpsandbox/nlpsandbox-schemas/blob/develop/openapi/commons/paths/healthCheck.yaml

**Review by tschaffter — CHANGES_REQUESTED** [review]

**Comment by tschaffter** [comment]
> I also moved the unit tests written for the models out of integration/ and into a unit/ folder.

@vpchung While our tests may look like unit tests because we have one test per endpoint, I still consider them as integration test. For example, if we test DELETE Challenge, it's possible in the future that the controller will perform actions other than simply delete the challenge object from the DB. For example DB objects related to the challenge that are no longer needed may also be deleted by calling other controllers' functions, an email notification may be sent to the challenge manager, etc. Therefore testing and thus calling DELETE Challenge is not testing a single unit of the server but test how well DELETE Challenge integrates into the codebase. A unit test would be the testing of a single class, for example testing a Model class.

**Comment by vpchung** [comment]
>A unit test would be the testing of a single class, for example testing a Model class.

Agreed, and that's exactly what I did :) The tests for the endpoints remain in `/integration`; I only moved the tests for the models into `/unit`.

**Comment by vpchung** [comment]
@tschaffter I'm not sure if a `404 Not Found` status code is needed for healthCheck?  Since we are not querying for a specific element within the database, I didn't think we would encounter this error and so, have removed it.

Unless there are some plans down the line where we will query a heathCheck on a specific model perhaps?

**Comment by tschaffter** [comment]
> I'm not sure if a 404 Not Found status code is needed for healthCheck? Since we are not querying for a specific element within the database, I didn't think we would encounter this error and so, have removed it.

I agree with you.

**Review by tschaffter — APPROVED** [review]
This PR looks great to me!

**Pull request merged**